src/Simplex/Chat/Controller.hs

```diff
@@ -752,8 +752,7 @@ data ChatResponse
   | CRUnknownMemberCreated {user :: User, groupInfo :: GroupInfo, forwardedByMember :: GroupMember, member :: GroupMember}
   | CRUnknownMemberBlocked {user :: User, groupInfo :: GroupInfo, blockedByMember :: GroupMember, member :: GroupMember}
   | CRUnknownMemberAnnounced {user :: User, groupInfo :: GroupInfo, announcingMember :: GroupMember, unknownMember :: GroupMember, announcedMember :: GroupMember}
-  | CRGroupEmpty {user :: User, groupInfo :: GroupInfo}
-  | CRGroupRemoved {user :: User, groupInfo :: GroupInfo}
+  | CRGroupEmpty {user :: User, groupId :: GroupId, groupName :: GroupName}
```

Comment:
also probably better to make a type for group reference

Reply:
ShortGroupInfo (see below)

Reply:
Then it should mirror Group/GroupInfo so in response it's without list of members. ShortGroup/ShortGroupInfo - latter in response